### PR TITLE
RavenDB-19006 Add Hub task details in ongoing task list view - in React

### PR DESCRIPTION
**Pull request by Danielle9897**
### Issue link
https://issues.hibernatingrhinos.com/issue/RavenDB-19006

### Additional description
Added details for Hub task in Tasks List view

### Type of change
- New feature

### How risky is the change?
- Low 

### Backward compatibility
- Non breaking change

### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing 
- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

File: src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx [hunk 1]
```diff
@@ -44,14 +35,31 @@ function Details(props: ReplicationHubPanelProps & { canEdit: boolean }) {
...
+                        Has Filtering:
+                        <div className="value">{data.shared.hasFiltering ? "True" : "False"}</div>
+                    </RichPanelDetailItem>
                 </RichPanelDetails>
             )}
 
             <div className="margin">
-                {connectedHubs.map((hub) => (
-                    <ReplicationHubPanel key={hub.shared.taskId + hub.shared.taskName} db={db} data={hub} />
+                {ongoingHubs.map((sink) => (
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx]: you map on hub but variable is sink?

Comment (ml054) [reply src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx]: and you pass sink to hub panel?

File: src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx [hunk 2]
```diff
@@ -44,14 +35,31 @@ function Details(props: ReplicationHubPanelProps & { canEdit: boolean }) {
...
+                    <ReplicationHubPanel key={sink.shared.taskId + sink.shared.taskName} db={db} data={sink} />
                 ))}
             </div>
+
+            <div className="margin">
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx:55]: why we can't fail fail like it was previously?

File: src/Raven.Studio/typescript/components/common/RichPanel.scss [hunk 3]
```diff
@@ -74,7 +74,6 @@
         padding: sizes.$gutter-xs math.div(sizes.$gutter, 2);
         padding-left: sizes.$gutter-md;
         display: flex;
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/common/RichPanel.scss:76]: why removed?

File: src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubDefinitionPanel.tsx]: rename: 
ongoingHubs => connectedSinks

File: src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubConnectedSinkPanel.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ interface ReplicationHubPanelProps {
     data: OngoingTaskReplicationHubInfo;
 }
 
-export function ReplicationHubPanel(props: ReplicationHubPanelProps) {
+export function ReplicationHubConnectedSinkPanel(props: ReplicationHubPanelProps) {
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubConnectedSinkPanel.tsx]: rename props type to match. There is convention there. If functional component is named x then props usually has type: xProps

Comment (Danielle9897) [reply src/Raven.Studio/typescript/components/pages/database/tasks/list/panels/ReplicationHubConnectedSinkPanel.tsx]: ok - good to know - done

**Review by ml054 — APPROVED** [review]

**Pull request merged**